docs/plugins/ingest-geoip.asciidoc

```diff
@@ -180,6 +180,33 @@ Which returns:
 --------------------------------------------------
 // TESTRESPONSE
 
+[[mappings-note]]
+===== Recognizing Location as a Geopoint
+Although this plugin enriches your document with a `location` field containing
+the estimated latitude and longitude of the ip-address, this field will not be
+indexed as a <<geo-point,`geo_point`>> type in Elasticsearch without explicitely defining it
+as such in the mapping.
+
+For example, here would be the mapping for the example index used in the above examples
+
+[source,js]
+--------------------------------------------------
+{
+  "mappings": {
+    "my_type": {
+      "properties": {
+        "geoip": {
+          "properties": {
+            "location": { "type": "geo_point" }
+          }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// NOTCONSOLE
```

Comment:
Can you make the example code a request to set mappings or create an index so this is tested?

Reply:
you make a reasonable request — will update.